lib/modules/ipfs/index.js

```diff
@@ -13,26 +13,48 @@ class IPFS {
     this.events = embark.events;
     this.buildDir = options.buildDir;
     this.storageConfig = embark.config.storageConfig;
+    this.namesystemConfig = embark.config.namesystemConfig;
     this.embark = embark;
 
     this.webServerConfig = embark.config.webServerConfig;
     this.blockchainConfig = embark.config.blockchainConfig;
 
-    if (!this.isIpfsEnabledInTheConfig()) {
-      return;
+    if (this.isIpfsEnabledInTheConfig() || this.isIpnsEnabledInTheConfig()) {
```

Comment:
IPFS and IPNS are so similar that I thought you called the same function twice. Is there a way to differentiate them?

Reply:
What do you think of this naming:
- `isIpfsEnabledInTheConfig` =>`isIpfsStorageEnabledInTheConfig`
- `isIpnsEnabledInTheConfig` => `isIpfsNameEnabledInTheConfig`

Does that work better for you?

Reply:
Yeah that sounds good!